app/views/layouts/application.html.erb

```diff
@@ -19,9 +19,12 @@
         <%= render 'breadcrumbs' %>
       <% end %>
     <% end %>
-
-    <main id="content" role="main" class="content <%= yield :page_class %>" <%= "lang=#{@content_item["locale"]}" unless !@content_item["locale"] || @content_item["locale"].eql?("en") %>>
-      <%= yield %>
+    <% if @content_item %>
+      <main id="content" role="main" class="content <%= yield :page_class %>" <%= "lang=#{@content_item["locale"]}" unless !@content_item["locale"] || @content_item["locale"].eql?("en") %>>
+    <% else %>
+      <main id="content" role="main" class="content <%= yield :page_class %>" <%= "lang=#{@lang}" unless !@lang == "en" %>>
```

Comment:
I'm not sure what this does right now! If we're only support `en` for now, can we simplify it?

Reply:
Yes! Done 👍